doc/classes/RigidBody2D.xml

```diff
@@ -113,7 +116,7 @@
 			<return type="float">
 			</return>
 			<description>
-				Returns the body's moment of inertia. Automatically computed from associated [class CollisionShape2D]s' mass during physic frames. Inertia is not computed the same frame in which the node was added in. Therefore inertia is not computed during the [code]_ready[/code] function.
+				Returns the body's moment of inertia. Computed automatically from the body's mass and assigned [Shape2D]\s during the physics frame. Note that it will not yet have a value in the [code]_ready()[/code] function.
```

Comment:
- `[Class]\ s` with a space between `\` and `s`
- should the first and second sentences be merged?